.github/workflows/pypipublish.yaml

```diff
@@ -0,0 +1,26 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools setuptools-scm wheel twine
+      - name: Build and publish
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
@martindurant - this workflow requires you to add secrets either in this repository specifically, or for the `intake` organization https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization

Reply:
I'd love to have the workflow eventually for all the repos, so I'll add it to the org.

Reply:
Added. This means that you should be able to merge and tag, and we'll get a new version. The resultant conda-forge PR will still need merging when it's ready.